### ISSUE TITLE
Past solutions inconsistency?

### ISSUE THREAD

**Issue by brendanvos** [Codewars/codewars.com#865]
Last night I completed [this kata](https://www.codewars.com/kata/how-many-numbers-iii).
- I am able to see the kata solutions
- I am able to see my "past solution" when I "train again"
- I see my solution on https://www.codewars.com/users/brendanvos/completed_solutions

Yet when I go to https://www.codewars.com/kata/how-many-numbers-iii/solutions/python/me/best_practice I am greeted with "There are no solutions yet for this kata."
_______
I am wondering if I perhaps forgot to submit (intending to refactor) after finding a valid solution. Nonetheless it's quite an odd inconsistency which may confuse users.

**Comment by brendanvos** [comment]
It happened again for [this kata](http://www.codewars.com/kata/strip-url-params/train/python). This time I'm certain I submitted my solution.

**Comment by suic86** [comment]
@jhoffner I clicked on [this](http://www.codewars.com/kata/strip-url-params/train/python) and I was surprisingly able to train without being logged in. Here's another example:

<img width="1226" alt="screen shot 2017-03-27 at 19 15 54" src="https://cloud.githubusercontent.com/assets/11985251/24368777/17285282-1322-11e7-8d9c-385bec263019.png">

Is this expected?

Regards,

suic

**Comment by brendanvos** [comment]
@suic86 I've been able to do that before as well. I figured that was expected behavior, you're simply not able to submit unless you've logged in. Seems like acceptable behavior, especially due to the fact that codewars will often log you out unexpectedly (which I often notice when I have many CW tabs open).

**Comment by jhoffner** [comment]
Yes it is expected. You are supposed to be able to submit as well. It's only when you go to view the solutions after completing do you need to signup.

**Comment by brendanvos** [comment]
@jhoffner what about the original issue?

**Comment by jhoffner** [comment]
Sorry didn't realize the actual issue was about something different. Not sure yet. I need to investigate. Only original solutions that were created before viewing all solutions (no forfeit) are shown in this list. If that's not the case this is probably a bug

**Comment by brendanvos** [comment]
@jhoffner I've never forfeited a kata, so I can assure you that's not the cause.

**Comment by brendanvos** [comment]
This seems to be related to some old issues which remain open: #122 and #49

**Comment by strah** [comment]
This happened to me today on [this kata](http://www.codewars.com/kata/imperfect-network-number-2-out-of-order-messages). **I am sure** I have submitted it ("Submit Final").
